.travis.yml

```diff
@@ -22,9 +18,8 @@ before_cache:
   - find $HOME/.sbt -name "*.lock" -print -delete
 env:
   global:
-    - PYTHON3_HOME=/opt/python/3.7.1
-    - PATH=$PYTHON3_HOME/bin\:$PATH
-  matrix:
+    - PATH=/opt/python/3.7.1/bin\:$PATH
+  jobs:
```

Comment:
what motivated the switch from `matrix` to `jobs`?

Reply:
While making the other build updates this was recommended by the [linter](https://config.travis-ci.com/explore).

From a [Travis CI 2019 blog post](https://blog.travis-ci.com/2019-10-24-build-config-validation#changes):

> One change that has been made intentionally is moving from matrix to jobs. So far, both the keys matrix and jobs were functionally equivalent, and jobs was an alias to matrix. We are now settling with jobs being the main, recommended key, and matrix being an alias for it. If you are using matrix in your build config you will see an informative validation message.